packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -82,7 +82,7 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 				} ) }
 			>
 				{ suggestion.is_free ? (
-					__( 'Free' )
+					__( 'Included in free plan' )
```

Comment:
i18n bot will come and ask for screenshots for this change. ;-)

Reply:
Attached